public/app/partials/reset_password.html

```diff
@@ -3,7 +3,14 @@
 <div class="page-container page-body">
 	<div class="signup">
 		<h3 class="p-b-1">Reset password</h3>
-		<form name="sendResetForm" class="login-form gf-form-group" ng-show="mode === 'send'">
+
+		<div ng-if="ldapEnabled || authProxyEnabled">
```

Comment:
For these conditionals to work you'll need to populate those props in the controller scope [here](https://github.com/grafana/grafana/blob/master/public/app/core/controllers/reset_password_ctrl.ts#L8) similar to this [code](https://github.com/grafana/grafana/blob/master/public/app/core/controllers/login_ctrl.ts#L22).